transforms/no-implicit-this/helpers/plugin.js

```diff
@@ -87,6 +87,16 @@ function doesTokenNeedThis(
     return true;
   }
 
+  // This is to support the ember-holy-futuristic-template-namespacing-batman syntax
```

Comment:
are many apps using this?

Reply:
I don't think so since it is still an experimental add-on. I thought the end goal is to start adopting this to avoid future collisions with [Nested Invocations in Angle Bracket Syntax](https://github.com/emberjs/rfcs/pull/457).  @rwjblue thoughts?